packages/framework/oldest-client-observer/package.json

```diff
@@ -86,8 +87,10 @@
 		"typescript": "~4.5.5"
 	},
 	"typeValidation": {
-		"disabled": true,
-		"version": "2.0.0-internal.2.1.0",
+		"version": "2.0.0-internal.3.2.0",
```

Comment:
This one should stay disabled - as an experimental package we're not making type compatibility promises yet (and it complicates the rename to `@fluidframework` scope as the type validator can't figure out the previous package).

We should enable typetests after publishing as non-experimental.

Reply:
OK; I'll remove the scripts as well. That's what made me think this was a mistake.

Reply:
That's ok too - I think they were added preemptively since we anticipate eventually turning them on, but no big deal to just add them at the appropriate time.